### PR TITLE
use amplify auth wrapper with cognito user pool

### PR DESCRIPTION
**Pull request by gorskilauren**
Also I swapped out parcel for `create-react-app` ,hence all the react svg/pngs .

### CODE REVIEW

File: ui/src/App.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import './App.css';
+import { Amplify } from "aws-amplify";
+import "@aws-amplify/ui-react/styles.css";
+import { Authenticator } from "@aws-amplify/ui-react";
+
+Amplify.configure({
+  Auth: {
+    userPoolId: 'us-east-2_SrK5dzmjJ',
```
Comment (gorskilauren) [inline ui/src/App.js:8]: we could throw these values in as github secrets (and re-create) when the actions are ready

File: cdk/src/RecordSpaStack.ts [hunk 2]
```diff
@@ -1,11 +1,40 @@
...
+            generateSecret: false,
+            oAuth: {
+                callbackUrls: [`https://${cloudfrontDomain}`],
+                flows: {
+                    authorizationCodeGrant: true
+                }
+            }
+        });
+
+        userPool.addClient('RecordSpaUser', {
```
Comment (kstasko) [inline cdk/src/RecordSpaStack.ts:30]: 🙊

File: cdk/src/StaticWebHost.ts [hunk 3]
```diff
@@ -44,14 +45,16 @@ export class StaticWebHost extends Construct {
...
             destinationBucket: recordSiteBucket,
             distribution: recordDistribution,
             distributionPaths: ["/*"],
             retainOnDelete: false
         });
 
-        new CfnOutput(this, 'cloufront domain name', { value: recordDistribution.distributionDomainName });
+        this.cloudfrontDomain = recordDistribution.distributionDomainName
+
+        new CfnOutput(this, 'cloufront domain name', { value: this.cloudfrontDomain });
```
Comment (kstasko) [inline cdk/src/StaticWebHost.ts:57]:
```suggestion
        new CfnOutput(this, 'cloudfront domain name', { value: this.cloudfrontDomain });
```

**Review by kstasko — APPROVED** [review]
👍

**Pull request merged**